packages/vertexai/src/requests/response-helpers.ts

```diff
@@ -196,3 +199,52 @@ export function formatBlockErrorMessage(
   }
   return message;
 }
+
+/**
+ * Convert a generic successful fetch {@link Response} body to an Imagen response object
```

Comment:
I don't think the link does anything if this is internal and won't go in the docs.

Reply:
I hadn't thought of that- I'll remove this.